src/components/Chat/InputMessageV2.tsx

```diff
@@ -36,7 +36,7 @@ const styles = StyleSheet.create({
   },
   container: {
     flex: 1,
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the background color of the container might affect readability and contrast depending on the colors of other components within this container. Please ensure that this color change is in line with accessibility standards, especially for users with visual impairments.
```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->